docs/source.rst

```diff
@@ -16,24 +16,26 @@ Source Guide
    communications with them. If you plan to use SecureDrop to maintain your
    anonymity, you should not discuss your own use of it with others via unsafe
    methods, including email to Freedom of the Press Foundation.
+   
+What is SecureDrop?
+---------------------------
+
+Dozens of news organizations — from *ProPublica* to *The New York Times* — use SecureDrop to accept tips securely and anonymously. You can reach out and share files, and messages, but for real anonymity, it’s important to take some extra precautions. This resource will describe things you can do to help protect your anonymity when using SecureDrop.
```

Comment:
Per our docs guidelines, would be good to format all new text to 80 characters width for consistency. I can add a commit to do that @martinshelton if you want, once all other comments have been resolved.